src/overlay/test/OverlayTests.cpp

```diff
@@ -1558,13 +1558,16 @@ TEST_CASE("overlay flow control", "[overlay][flowcontrol]")
 
     auto configs = std::vector<Config>{};
 
+    uint32_t const floodReadingCapacity = 1;
```

Comment:
This commit is not needed because of the asserts in the second commit (so the test should fail regardless)

Reply:
Removed.